cylc/flow/network/__init__.py

```diff
@@ -109,15 +109,15 @@ def deserialize(message: str) -> 'ResponseDict':
     return json.loads(message)
 
 
-def get_location(workflow: str) -> Tuple[str, int, int]:
+def get_location(workflow: str) -> Tuple[str, int, int, str]:
     """Extract host and port from a workflow's contact file.
 
     NB: if it fails to load the workflow contact file, it will exit.
 
     Args:
         workflow: workflow ID
     Returns:
-        Tuple (host name, port number, publish port number)
+        Tuple (host name, port number, publish port number, scheduler version)
```

Comment:
Enable Tui to see the Scheduler version (before issuing a GraphQL query) for back-compat purposes - https://github.com/cylc/cylc-flow/pull/6671/commits/a995c29a50daef369bf1095a6a19b21c369b99e9